content/en/docs/tasks/observability/distributed-tracing/configurability/index.md

```diff
@@ -55,6 +55,40 @@ always provided in the annotations to ensure that the traces are reported
 correctly for the workload.
 {{< /warning >}}
 
+## Installation
+
+Using these features opens new possibilities for managing traces in your environment.
+
+In this example, we will sample all traces and add a tag named `clusterID`
+using the `ISTIO_META_CLUSTER_ID` environmental variable injected into your pod. Only the
+first 256 characters of the value will be used if its length is higher.
+
+{{< text bash >}}
+$ cat <<EOF > ./tracing.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      tracing:
+        sampling: 100.0
+        max_path_tag_length: 256
+        custom_tags:
+          clusterID:
+          environment:
+            name: ISTIO_META_CLUSTER_ID
+EOF
+$ istioctl install --set values.meshConfig.enableTracing=true -f ./tracing.yaml
+{{< /text >}}
+
+{{< tip >}}
+We also had to make sure to set `meshConfig.enableTracing=true` during installation. This can
+done as a part of `tracing.yaml`.
+{{< /tip >}}
+
+This document will outline how to build and configure your own configuration options and their
```

Comment:
This sentence doesn't seem correct. Maybe `This document will outline the configuration options, their meanings and how to build your own configuration`, but not sure that's exactly what you are trying to say.

Reply:
Removed this section